src/nevermined/Files.ts

```diff
@@ -13,8 +13,8 @@ export class Files extends Instantiable {
         return instance
     }
 
-    public async uploadFilecoin(stream: ReadStream, encrypt?: boolean): Promise<any> {
-        const response = await this.nevermined.node.uploadFilecoin(stream, encrypt)
+    public async uploadFilecoin(data: ReadStream | string, encrypt?: boolean): Promise<any> {        
```

Comment:
which type return this method? cannot it be specify?

Reply:
this comes from `node-fetch` that returns an `any`